bosh/releases/pre_render_scripts/diego-cell/cflinuxfs3-rootfs-setup/jobs/patch_pre-start.sh

```diff
@@ -17,3 +17,11 @@ patch --verbose "${target}" <<'EOT'
  TRUSTED_CERT_FILE=$CONF_DIR/certs/trusted_ca.crt
  CA_DIR=$ROOTFS_DIR/usr/local/share/ca-certificates/
 EOT
+)
+
+# Only patch once
+if ! patch --reverse --dry-run -f "${target}" <<<"$PATCH" 2>&1  >/dev/null ; then
+  patch --verbose "${target}" <<<"$PATCH"
+else
+  echo "Patch already applied. Skipping"
+fi
```

Comment:
[nit] missing a line break.

Reply:
Done. It would be great to get this finally merged. Just let us know if there is still something open.